### ISSUE TITLE
Feature Request: Sharing Lists or Tags

### ISSUE THREAD

**Issue by FanyangMengDev** [Kovah/LinkAce#146]
Hi Kovah,

Would you please add this feature?

Actually sometimes we need to share the URLs in the same list, which means the most convenient way is sharing the whole list.

If we can share the lists or tags. I think LinkAce could be promoted more extensively.

**Comment by Kovah** [comment]
Could you explain more about this feature or give an example on how it should work? I think I do not fully understand what you are trying to achieve. Should the list be shared via social media buttons like links? Maybe you could describe your problem.

**Comment by Kovah** [comment]
Any news on this? Could really need some feedback on this.

**Comment by FanyangMengDev** [comment]
Sorry for the delay, I forgot to reply you last time.

So what I mean is, we can share a list or a tag's link no matter what we set if the linkace allows the guests. Once we share it, and people open this link, they can see all the links in it, even if they are guests.

I send this feature request is because I forbidden the guest for my site first, then after I turn on it again, and share the list link to my friend, the link returns an error as the following:

**Comment by Kovah** [comment]
Okay sure, I understand. I think this is not a feature that will make it into 1.0, but I will keep it for a later version.

About the error: Did you run the migrations after upgrading to a newer version?

**Comment by FanyangMengDev** [comment]
> Okay sure, I understand. I think this is not a feature that will make it into 1.0, but I will keep it for a later version.
> 
> About the error: Did you run the migrations after upgrading to a newer version?

Yes, I have already run the migrations and it displayed nothing to migrate. Really weird.

**Comment by amadeusp** [comment]
Personally, I would not consider this to be an important feature to me but I could think of cases were it could come in handy to be able to share tags/lists with someone with a simple link. I would eventually want to limit the link in validity, e.g. for a specific amount of days or times accessed.

**Comment by Kovah** [comment]
Pushed this to later, too much work right now to bring it to the v2 release.
